### ISSUE TITLE
Why is the policy loss after the minus sign?

### ISSUE THREAD

**Issue by kennychenfs** [lightvector/KataGo#82]
Loss=winrate loss-policy loss+L2 loss
Why?
and why there are log() in loss function?
What are their use?

**Comment by OmnipotentEntity** [comment]
Please read this article for the reason. https://machinelearningmastery.com/loss-and-loss-functions-for-training-deep-learning-neural-networks/

**Comment by lightvector** [comment]
@OmnipotentEntity - thanks for answering, I had missed this issue somehow.

@kennychenfs - I'm going ahead and closing this issue as this is a very basic machine learning/optimization question, rather than anything to do with KataGo.  If you do have further questions like this, if you think they might be Go-bot-specific, feel free to ask them in the LZ discord such as in the "#noob" channel ( https://discord.gg/wrnERjQ ) or if they are very general questions, you can try places like here: https://www.reddit.com/r/MLQuestions/.

**Comment by kennychenfs** [comment]
I have some ML basic, but I just forgot how the cross-entropy loss works.

**Comment by kennychenfs** [comment]
thanks for your answer
